go.mod

```diff
@@ -2,45 +2,60 @@ module github.com/filhodanuvem/ytgoapi
 
 go 1.21.0
 
+require (
+	github.com/gin-gonic/gin v1.9.1
+	github.com/google/uuid v1.3.1
+	github.com/jackc/pgx/v4 v4.18.1
+	github.com/jackc/pgx/v5 v5.4.3
```

Comment:
eu to um pouco incomodado com esse import do v4 e v5 ao mesmo tempo, acho que eu importei as duas libs em arquivos. Se tiver tempo pode por favor trocar todas as mencoes de /v5 por /v4 e rodar um `go mod tidy`? 

Tem uma outra PR tentando resolver o mesmo caso #8 mas se conseguir encontrar o problema, mas ainda nao encontramos esse problema la.

Reply:
Posso ver, mas acho que seria melhor abrir um pr separado

Reply:
feito o merge e resolvido os conflitos